src/app/clusters/operational-credentials-server/operational-credentials-server.cpp

```diff
@@ -59,6 +60,24 @@ namespace {
 constexpr uint8_t kDACCertificate = 1;
 constexpr uint8_t kPAICertificate = 2;
 
+CHIP_ERROR CreateAccessControlEntryForNewFabricAdministrator(FabricIndex fabricIndex, NodeId subject)
+{
+    Access::AccessControl::Entry entry;
+    ReturnErrorOnFailure(Access::GetAccessControl().PrepareEntry(entry));
+    ReturnErrorOnFailure(entry.SetFabricIndex(fabricIndex));
+    ReturnErrorOnFailure(entry.SetPrivilege(Access::Privilege::kAdminister));
+    ReturnErrorOnFailure(entry.SetAuthMode(Access::AuthMode::kCase));
+    ReturnErrorOnFailure(entry.AddSubject(nullptr, subject));
+    ReturnErrorOnFailure(Access::GetAccessControl().CreateEntry(nullptr, entry));
+
+    emberAfPrintln(EMBER_AF_PRINT_DEBUG, "OpCreds: ACL entry created for Fabric %" PRIX8 " CASE Admin NodeId 0x" ChipLogFormatX64,
+                   fabricIndex, ChipLogValueX64(subject));
+
+    // TODO: event notification for newly created ACL entry
```

Comment:
I woudl think CreateEntry would do that automatically...

Reply:
Events are a cluster level thing. This is the only place not in the cluster that manages an entry. While we could add a listener from the module to the cluster (and that might be how we do it), we could also do it with another mechanism. Hence the TODO. Either way, it needs to be done.

Reply:
Sure.  My point is that ideally whatever API allows creating entries would also handle creating the right events, instead of relying on all consumers to get the sequencing right.....